src/library_fs.js

```diff
@@ -290,6 +290,12 @@ FS.staticInit();
       if (errCode) {
         throw new FS.ErrnoError(errCode);
       }
+      if (name === "..") {
+        return parent.parent;
+      }
+      if (name === ".") {
+        return parent;
+      }
```

Comment:
If this is required here, how have we gotten away with not having it up until now?

Reply:
I don't think it's required since we always filter out the "." components.